devel/orcus/Portfile

```diff
@@ -24,7 +24,7 @@ checksums           rmd160  8c18cf3459d8054dd36215db16c99720dcc55a3c \
                     size    7945943
 
 use_autoreconf      yes
-configure.cxxflags-append -std=c++14
+configure.cxxflags-append -std=gnu++17
```

Comment:
The c++ standard needs to be specified with a line like

```
compiler.cxx_standard 2017
```

That should also automatically add the cxxflag, so it likely will no longer be required.

Reply:
It needs both to work as far as I can tell.

Reply:
Specifying `compiler.cxx_standard` does not automatically add any `-std` flag. It is expected that the project's build system will do so. If it does not, we add it manually in the portfile.